pkg/kubemark/hollow_proxy.go

```diff
@@ -82,7 +82,7 @@ func NewHollowProxyOrDie(
 	if useRealProxier {
 		nodeIP := utilnode.GetNodeIP(client, nodeName)
 		if nodeIP == nil {
-			klog.V(0).Infof("can't determine this node's IP, assuming 127.0.0.1")
+			klog.V(0).InfoS("Can't determine this node's IP, assuming 127.0.0.1")
```

Comment:
Why InfoS if there are no key values?

Reply:
We can mark directory as migrated in static analysis.